### PR TITLE
Allow changing expected Authorization header_prefix in subclasses of JwtAuthentication

### LINKED ISSUE #83: Allow Specifying the Expected Scheme in JwtAuthentication for Authorization Header
Currently it is hardcoded as `Bearer`.

### PR DESCRIPTION
**Pull request by cancan101**
Closes #83

### CODE REVIEW

**Comment by cancan101** [comment]
@taion updated.

File: flask_resty/jwt.py [hunk 1]
```diff
@@ -26,6 +26,7 @@ class JwtAuthentication(AuthenticationBase):
     CONFIG_KEY_TEMPLATE = 'RESTY_JWT_DECODE_{}'
 
     id_token_arg = 'id_token'
+    header_scheme = 'Bearer'
```
Comment (taion) [inline flask_resty/jwt.py]: Move this above `id_token_arg` (since it's used first in the code below)

**Comment by taion** [comment]
One more nit, then this is GTG

**Comment by cancan101** [comment]
Hood to merge?

**Pull request merged**